docs/userGuide/syntax/annotations.md

```diff
@@ -0,0 +1,145 @@
+  <!-- Customize Point Header (default is empty) -->
+  <a-point x="75%" y="25%" content="Lorem ipsum dolor sit amet"  header="Lorem ipsum"/>
+  <!-- Customize Point Color (default color is green) -->
+  <a-point x="33%" y="50%" content="Lorem ipsum dolor sit amet"  color="red"/>
+  <!-- Customize Point Opacity (default opacity is 0.3) -->
+  <a-point x="66%" y="50%" content="Lorem ipsum dolor sit amet"  opacity="0.7"/>
+  <!-- Customize Point Label (default is empty) -->
+  <a-point x="25%" y="75%" content="Lorem ipsum dolor sit amet" label="1"/>
+  <!-- Customize Text Color (default color is black) -->
+  <a-point x="50%" y="75%" content="Lorem ipsum dolor sit amet"  textColor="white" color="black" label="2" opacity="1"/>
+  <!-- Customize Font Size (default font size is 14) -->
+  <a-point x="75%" y="75%" content="Lorem ipsum dolor sit amet"  fontSize="30" label="3"/>
+</annotate>
+</variable>
+</include>
+
+**Using trigger for Annotate Point** <br>
+
+Similar to **[popovers](popups.md)**, annotation points also support different types of triggers
+for users with different needs.
+
+<include src="codeAndOutput.md" boilerplate >
+<variable name="highlightStyle">html</variable>
+<variable name="code">
+
+<annotate src="../../images/annotateSampleImage.png" width="500" alt="sampleImage">
+  <!-- Default Trigger (click)-->
+  <a-point x="33%" y="33%" content="Lorem ipsum dolor sit amet" />
+  <!-- Set Trigger to hover focus -->
+  <a-point x="66%" y="33%" content="Lorem ipsum dolor sit amet" trigger="hover focus"/>
+  <!-- Set Popover Placement (click)-->
+  <a-point x="25%" y="66%" content="Lorem ipsum dolor sit amet" placement="left"/>
+  <a-point x="50%" y="66%" content="Lorem ipsum dolor sit amet" placement="bottom"/>
+  <a-point x="75%" y="66%" content="Lorem ipsum dolor sit amet" placement="right"/>
+</annotate>
+</variable>
+</include>
+
+<br>
+
+**Displaying content as legends in Annotate Point** <br>
+
+Annotate point also allows users to display its content inside the popover or as
+a legend below the diagram or both. However, this only works when if a label is
+also provided for the point.
+
+<include src="codeAndOutput.md" boilerplate >
+<variable name="highlightStyle">html</variable>
+<variable name="code">
+
+<annotate src="../../images/annotateSampleImage.png" width="500" alt="sampleImage">
+  <!-- Default Legend (popover only)-->
+  <a-point x="25%" y="50%" content="There is only text when you click me" label="1"/>
+  <!-- Set Legend to bottom only (popover is not clickable) -->
+  <a-point x="50%" y="50%" content="Clicking on this does nothing" label="2" legend="bottom" header="Headers are displayed as well"/>
+  <!-- Set Legend to both -->
+  <a-point x="75%" y="50%" content="There is text at both locations"  label="3" legend="both" header="Headers are displayed at both positions"/>
+</annotate>
+</variable>
+</include>
+
+<br>
+
+****`<a-point>` Options****
```

Comment:
Can consider using a `###`, so that the pageNav is utilized, same for the annotate options. Currently the pageNav is not very meaningful:
![image](https://user-images.githubusercontent.com/41845017/163722078-78821807-6d21-45e7-9ba5-672fc578d190.png)

Reply:
This style is in line with every other option style so I just followed the existing format

Reply:
In that case perhaps can consider removing the "introduction" title since there is nothing else besides it. Also can consider using a list here for clarity:
So instead of 
![image](https://user-images.githubusercontent.com/41845017/163741552-146527c8-8503-49e2-8177-53a383048154.png)

use

## Annotate
Annotate components allow you to easily annotate over any image.

- `<annotate>`: Annotate wrappers are used to hold the image and set its width and height.
-  `<a-point>`: Annotate points define the position, text and style of each point within the image. Insert them between the Annotate wrappers.